anomdtct/anomdtct/pipeline.py

```diff
@@ -0,0 +1,151 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import timedelta, date
+import pandas as pd
+from google.cloud.bigquery._helpers import _bytes_to_json
+from google.cloud import bigquery
+from google.cloud.exceptions import NotFound
+from anomdtct.utils import s2d
+from anomdtct.data import get_raw_data, prepare_data, get_data_for_date, prepare_training_data
+from anomdtct.forecast import forecast, fit_model
+from anomdtct.output import write_records, write_to_spreadsheet, write_model
+import logging
+
+
+DEFAULT_BQ_PROJECT = "moz-fx-data-shared-prod"
+DEFAULT_BQ_DATASET = "analysis"
+DEFAULT_BQ_TABLE = "deviations"
+
+DEFAULT_BQ_MODEL_CACHE_PROJECT = "moz-fx-data-shared-prod"
+DEFAULT_BQ_MODEL_CACHE_DATASET = "analysis"
+DEFAULT_BQ_MODEL_CACHE_TABLE = "deviations_model_cache"
```

Comment:
The production pipeline should be writing into tables in the `telemetry_derived` dataset and loading cached models from `telemetry_derived`. We can do this now since this is going to be run via Airflow and have the necessary creds.

I would like to see the production defaults written in here, and users running this locally would be expected to provision their own tables in the DS project or in `analysis` and configure the job accordingly.